_posts/2015-06-23-hardly-anyone-knows-cd.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Hardly Anyone Knows Continuous Delivery
+status: public
+type: post
+published: true
+author: Ken Mugrage
+---
+
+####Those of us who work in or around teams doing continuous delivery often think it’s a mainstream thing. This couldn’t be further from the truth.
+
+I work for ThoughtWorks, a company that implements processes and technologies we think are good long before most. We built the first CI server with Cruise Control, and Go was the first purpose built Continuous Delivery server. I go to a lot of conferences and events, read a lot of blogs, talk to a lot of peers, work with a lot of partners, etc. I thought most people involved with the creation of software had a pretty good idea what CD is.
+
+I was wrong.
+
+I just got back from a pretty big software conference that was a bit off my normal track. They had a few DevOps sessions this year, but historically this particular conference has been more about agile methodologies. As a sponsor of the event, I spent most of my time standing in the booth talking to people.
```

Comment:
Rephrase as - "As one of the sponsors, I spent a lot of time at the booth talking to people"